src/SearchComponents/SearchForm.svelte

```diff
@@ -51,6 +67,9 @@
         const representations = form["ResultsRepresentation"];
         let s = "";
         for (const key in representations) { //all entries in this dict are arrays
+            if (key === 'MultipleImages') {
+                s += '?mainres knora-api:hasStillImageFileValue ?imgfile .\n';
```

Comment:
to avoid adding this line multiple times to the search string, a break when exiting the if loop should be added